core/src/main/java/me/mykindos/betterpvp/core/effects/listeners/EffectListener.java

```diff
@@ -227,7 +227,8 @@ public void onDamage(CustomDamageEvent event) {
         Optional<Effect> effectOptional = effectManager.getEffect(event.getDamagee(), EffectType.VULNERABILITY);
         effectOptional.ifPresent(effect -> {
             if (event.getCause() == EntityDamageEvent.DamageCause.LIGHTNING) return;
-            event.setDamage((event.getDamage() * (1.0 + (effect.getLevel() * 0.25))));
+            double damageIncrease = 0.1 + (effect.getLevel() - 1) * 0.10;
```

Comment:
because this isn't a vanilla effect, you shouldnt do -1

You could keep it the same as it was before but replace 0.25 with 0.10